govlearn-api/src/main/java/com/unimuenster/govlearnapi/course/controller/CourseFilteringController.java

```diff
@@ -28,12 +33,15 @@ public class CourseFilteringController {
     private final ControllerCourseMapper controllerCourseMapper;
 
     @Operation(
-            description = "Filter courses by attributes."
+        security = { @SecurityRequirement(name = "Authorization") },
+        description = "Filter courses by attributes and categories."
     )
-    @GetMapping("/{name-search}")
-    public ResponseEntity<Response> filterCourses(@PathVariable("name-search") String nameSearch) {
+    @PreAuthorize("hasAuthority('user')")
+    @PostMapping("/{name-search}")
+    public ResponseEntity<Response> filterCourses(@PathVariable("name-search") String nameSearch, 
+        @RequestBody List<Long> categoryIDs) {
```

Comment:
Vielleicht ist es hier einfacher wenn man keine Liste an IDs sondern an Category-Namen übergibt, ist aber auch wieder von der Lösung im Frontend abhängig. @comlit was meinst du?

Reply:
Habe das nochmal angepasst. Die Suche läuft jetzt nur noch über die Tags. Heißt: Die Kategorien werden im Frontend in die Tags übersetzt